src/browser/commands/assert-view/index.js

```diff
@@ -155,7 +155,23 @@ module.exports.default = browser => {
         testplaneCtx.assertViewResults.add({ stateName: state, refImg: refImg });
     };
 
-    session.addCommand("assertView", async function (state, selectors, opts = {}) {
+    session.addCommand("assertView", async function (state, selectorsOrOpts, opts = {}) {
+        let selectors;
+        if (selectorsOrOpts) {
+            if (typeof selectorsOrOpts === "string" || _.isArray(selectorsOrOpts)) {
+                selectors = selectorsOrOpts;
+            } else if (typeof selectorsOrOpts === "object") {
+                opts = selectorsOrOpts;
+            }
+        }
+        if (!selectors) {
+            opts = _.defaults(opts, {
+                allowViewportOverflow: true,
```

Comment:
Why it is specified as `true` by default?

Reply:
Because without it we will get an error if we try to screen the viewport on a page with height of more than 1 screen.